security/logging/requests/utils.py

```diff
@@ -16,12 +16,6 @@ def is_base_collection(v):
     return isinstance(v, (list, tuple, set))
 
 
-def get_headers(request):
-    regex = re.compile('^HTTP_')
-    return dict((regex.sub('', header), value) for (header, value)
-                in request.META.items() if header.startswith('HTTP_'))
```

Comment:
before django 2.2 request.headers was not available. Unfortunatelly content typa and length is returned with this function